DOCSBUILD.md

```diff
@@ -0,0 +1,347 @@
+# Documentation Build Process
+
+This doc (from James Eddy) describes the build process for Swagger UI and static docs (in Travis CI) and subsequent deployment to GitHub Pages (cc @briandoconnor, @denis-yuen, @david4096, @natanlao).
+
+These instructions are based on the current configuration for the [**Workflow Execution Service (WES) API schema repo**](https://github.com/ga4gh/workflow-execution-service-schemas), which includes:
+
++ **When code is merged into the `master` branch of this repository**, artifacts are created and hosted at the following paths:
+  + ga4gh.github.io/[repo]/swagger-ui/ — Swagger UI for the API spec
+  + ga4gh.github.io/[repo]/docs/ — reference docs for the API
+  + ga4gh.github.io/[repo]/swagger.json — API spec in JSON format
+  + ga4gh.github.io/[repo]/swagger.yaml — API spec in YAML format
+
++ **For non-`master` branches**, reviewers can preview documentation and other pages under "ga4gh.github.io/[repo]/preview/[branch-name]/"
+  + swagger-ui/ — Swagger UI preview for current version of [branch-name]
+  + docs/ — docs preview for current version of [branch-name]
+  + swagger.json — spec (JSON) preview for current version of [branch-name]
+  + swagger.yaml — spec (YAML) preview for current version of [branch-name]
+
++ When changes are pushed to branches on a fork of the main repo (and the user has set up Travis for their forked repo), the same path apply but should be relative to "[user-or-org].github.io/[repo]/".
+
++ `README.md` and `CONTRIBUTING.md` updated with all of the above links
+
++ `README.md` badges indicating Travis CI build status and Swagger/OpenAPI validation status
+
+**Note:** This set up was the result of substantial trial and error. It includes tools/solutions based in languages I don't normally use (Gradle, Node). There should be plenty of room for improvement and cleanup — so suggestions are welcome!
```

Comment:
Not sure if you want to include my editorial notes in the these docs. :)

Reply:
OK, removed